src/components/EventCard.tsx

```diff
@@ -120,24 +29,78 @@ const EventCard: React.FC<IProps> = ({
   location,
   isEven,
 }) => (
-  <StyledCard className={className}>
-    <StyledCardReference />
-    <StyledCardContent
-      className={isEven ? "style-card-content-right" : undefined}
+  <Box
+    className={className}
+    position="relative"
+    marginTop={{ base: "30px", md: 0 }}
+    _before={clearStyles}
+    _after={clearStyles}
+  >
+    <Box
+      w="24px"
+      h="24px"
+      position="absolute"
+      top="0"
+      left="50%"
+      overflow="hidden"
+      marginLeft="-12px"
+      backgroundColor="primary"
+      display={{ base: "none", md: "block" }}
+    />
+    <Box
+      width={{ base: "100%", md: "45%" }}
+      padding={6}
+      backgroundColor="ednBackground"
+      borderRadius="2px"
+      border="1px solid"
+      borderColor="lightBorder"
+      float={isEven ? "right" : { base: "right", md: "none" }}
+      marginTop={isEven ? { base: 0, md: "-25%" } : 0}
+      _before={{
+        content: '""',
+        position: "absolute",
+        top: "10px",
+        width: 0,
+        height: "3px",
+        display: { base: "none", md: "inline" },
+        ...(isEven
+          ? {
+              left: "inherit",
+              right: "45%",
+              borderLeft: 0,
+              borderRight: "25px solid",
+            }
+          : {
+              left: "45%",
+              borderLeft: "25px solid",
+              borderRight: 0,
+            }),
+        borderColor: "primary",
+      }}
     >
-      <Date>
+      <Text color="primary" marginBottom={0} textAlign="right">
         {date}
-        <Emoji text=":spiral_calendar:" size={1} ml={`0.5em`} />
-      </Date>
-      <Location>
-        <LocationText>{location}</LocationText>
-        <Emoji text=":round_pushpin:" size={1} ml={`0.5em`} />
-      </Location>
-      <Title>{title}</Title>
-      <Description>{description}</Description>
+        <Emoji text=":spiral_calendar:" fontSize="md" marginLeft="0.5em" />
+      </Text>
+      <Text marginBottom={0} textAlign="right">
+        <Text as="span" opacity={0.6}>
+          {location}
+        </Text>
+        <Emoji text=":round_pushpin:" fontSize="md" marginLeft="0.5em" />
+      </Text>
+      <Heading
+        as="h3"
+        marginTop={0}
+        fontSize="2xl"
```

Comment:
We can ignore the size for this `h3` for now since we have added those sizes to the chakra global styles files (we will clean that up when we finish with the migration).
```suggestion
```